rusk/Cargo.toml

```diff
@@ -52,19 +52,21 @@ sha2 = "0.9"
 tonic-build = "0.3"
 rustc_tools_util = "0.2"
 dusk-bytes = "0.1"
-dusk-plonk = { version = "0.8", default-features = false, features = ["canon"] }
-dusk-poseidon = { version = "0.21.0-rc", features = ["canon"] }
+dusk-plonk = { version = "0.8", features = ["canon", "std"] }
+dusk-poseidon = {version="0.21.0-rc", features=["canon"]}
```

Comment:
I believe the last agreement on Cargo format is to put spaces. @ZER0 to confirm

Reply:
kk will add them